docs/markdown/Helm/helm-overview.md

```diff
@@ -113,6 +113,44 @@ Built Helm chart artifact: testprojects.src.helm.example/example/example-0.2.0.t
 
 The final output folder can customised using the `output_path` field in the `helm_chart` target. Run `pants help helm_chart` for more information.
 
+### Helm chart version
+
+Helm charts are versioned artifacts with the value of the `version` field in `Chart.yaml` determining the actual version of the chart. Pants needs to know the version of a first party chart to be able to build packages and correctly stablish the dependencies among them. By default, Pants will use the value in `Chart.yaml` as the given version of a chart but it also supports overring that value via the `version` field in the `helm_chart` target.
+
+For example, a chart defined as such:
+
+```python src/helm/example/BUILD
+helm_chart()
+```
+```yaml src/helm/example/Chart.yaml
+apiVersion: v2
+description: Example Helm chart
+name: example
+version: 0.1.0
+```
+
+Will be understood to have version `0.1.0` (as read from the Chart.yaml) file. However, if we specify a version in `helm_chart` as follows:
+
+```python src/helm/example/BUILD
+helm_chart(version="1.0.0")
+```
+```yaml src/helm/example/Chart.yaml
+apiVersion: v2
+description: Example Helm chart
+name: example
+version: 0.1.0
+```
+
+Now the value in `Chart.yaml` will be ignored and the chart will be understood to have version `1.0.0`.
```

Comment:
```suggestion
Now the value in `Chart.yaml` will be ignored and the chart will be understood to have version `2.0.0`.
```

Suggest to make the difference bigger than merely moving the `1` as that is a rather subtle change.

(couldn't keep it as a single suggestion as the diff contains tripple backticks messing with the suggested change box)